Servers/opc-ua/basic/deployment/config/opc-ua-server-securitypolicy.json

```
@@ -0,0 +1,27 @@
+{
+    "Library": "libxrt-opc-ua-server.so",
+    "Factory": "xrt_opc_ua_server_factory",
+    "ThreadPool": "pool",
+    "Bus": "bus",
+    "Logger": "logger",
+    "ApplicationUri" : "${XRT_OPC_UA_APPLICATION_URI}",
+    "SecurityPolicies": [
+    {
+      "SecurityPolicy": "Basic256Sha256",
+      "Certificate": "${XRT_OPC_UA_X509_CERTIFICATE}",
+      "PrivateKey": "${XRT_OPC_UA_X509_KEY}"
+    },
+    {
+      "SecurityPolicy": "None"
+    },
+  ],
+  "Endpoints": [
+    {
+      "SecurityPolicy": "Basic256Sha256",
+      "MessageSecurityModes" : ["Sign"]
+    },
+    {
+      "SecurityPolicy": "None"
+    }
```

Is it worth changing this to 
```suggestion
{
     "SecurityPolicy": "None",
     "DiscoveryOnly" : true
}
```
So that with a client they can only connect with security policy enabled

---

@dpazj, 'DiscoveryOnly" option is added only in 2.3 and not in 2.2

---

updated